src/main/java/taxi/controller/ucp/RemoveCarController.java

```diff
@@ -0,0 +1,28 @@
+package taxi.controller.ucp;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.model.Driver;
+import taxi.service.CarService;
+import taxi.service.DriverService;
+
+public class RemoveCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Car car = carService.get(Long.valueOf(req.getParameter("id")));
+        Driver driver = driverService.get((Long) req.getSession().getAttribute("driver_id"));
+        carService.removeDriverFromCar(driver, car);
+        resp.sendRedirect(req.getContextPath() + "/ucp");

```

Comment:
what is ucp? let's name this controller RemoveDriverFromCarController and place in `car` folder